content/blog/lift-and-transform-apps-with-hpe-csi-driver-for-kubernetes.md

```diff
@@ -0,0 +1,283 @@
+
+The TL;DR version of this blog post is available as an instructional lightboard video [published on YouTube](https://www.youtube.com/watch?v=M7t_qPe3i5E). The blog concretizes what’s being outlined in the lightboard video. 
+
+[![YouTube](/img/untitled.jpg)](https://www.youtube.com/watch?v=M7t_qPe3i5E)
+
+Still want the details? Let’s get started!
+
+# Origin state
+
+In the following examples we’re going to use a MariaDB database with the blessed [Employees database](https://github.com/datacharmer/test_db) as an example workload. It provides simple means to validate its contents to ensure the contents are intact throughout its journey.
+
+The database is running in a standalone VM hosted on a legacy NFS datastore. 
+
+![MariaDB running on VMware vSphere](/img/slide1.png "MariaDB running on VMware vSphere")
+
+The content is simply validated by loading a SQL statement file provided in the GitHub repo.
+
+```markdown
+$ mysql -u mmattsson -ppassword < test_employees_sha.sql
+INFO
+TESTING INSTALLATION
+table_name	expected_records	expected_crc
+departments	9	4b315afa0e35ca6649df897b958345bcb3d2b764
+dept_emp	331603	d95ab9fe07df0865f592574b3b33b9c741d9fd1b
+dept_manager	24	9687a7d6f93ca8847388a42a6d8d93982a841c6c
+employees	300024	4d4aa689914d8fd41db7e45c2168e7dcb9697359
+salaries	2844047	b5a1785c27d75e33a4173aaa22ccf41ebd7d4a9f
+titles	443308	d12d5f746b88f07e69b9e36675b6067abb01b60e
+table_name	found_records   	found_crc
+departments	9	4b315afa0e35ca6649df897b958345bcb3d2b764
+dept_emp	331603	d95ab9fe07df0865f592574b3b33b9c741d9fd1b
+dept_manager	24	9687a7d6f93ca8847388a42a6d8d93982a841c6c
+employees	300024	4d4aa689914d8fd41db7e45c2168e7dcb9697359
+salaries	2844047	b5a1785c27d75e33a4173aaa22ccf41ebd7d4a9f
+titles	443308	d12d5f746b88f07e69b9e36675b6067abb01b60e
+table_name	records_match	crc_match
+departments	OK	ok
+dept_emp	OK	ok
+dept_manager	OK	ok
+employees	OK	ok
+salaries	OK	ok
+titles	OK	ok
+computation_time
+00:00:49
+summary	result
+CRC	OK
+count	OK
+```
+
+In the remaining tests throughout this blog, only the summary results will be displayed.
+
+# Destination state
+
+When the transition has completed, the Employees database should be running on a Kubernetes cluster using persistent storage provided by the HPE CSI Driver for Kubernetes backed by a HPE Nimble Storage array. The procedures are similar regardless of backend, HPE Alletra and Primera included.
+
+![MariaDB running on Kubernetes](/img/slide2.png "MariaDB running on Kubernetes")
+
+While the destination state is a virtualized Kubernetes cluster within the same vSphere environment, that is not a requirement, it can be any hypervisor, bare-metal or otherwise as long as the destination cluster is compatible with the HPE CSI Driver for Kubernetes and any of the supported backends.
```

Comment:
Recommend modifying the beginning to:

"While the destination state in our example is a virtualized Kubernetes cluster..."

Place a period after "that is not a requirement". Make the next sentence: "It can be any hypervisor, bare-metal, or other as long as the destination cluster..."